media-libs/webvfx/webvfx-1.2.0.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic qmake-utils toolchain-funcs
+
+DESCRIPTION="Video effects library based on web technologies"
+HOMEPAGE="https://github.com/mltframework/webvfx/"
+SRC_URI="https://github.com/mltframework/${PN}/releases/download/{$PV}/${P}.txz"
```

Comment:
```
>>> Emerging (1 of 1) media-libs/webvfx-1.2.0::testworld
>>> Downloading 'https://github.com/mltframework/webvfx/releases/download/{1.2.0}/webvfx-1.2.0.txz'
--2020-07-09 07:21:32--  https://github.com/mltframework/webvfx/releases/download/%7B1.2.0%7D/webvfx-1.2.0.txz
Resolving github.com... 140.82.118.3
Connecting to github.com|140.82.118.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2020-07-09 07:21:33 ERROR 404: Not Found.

!!! Couldn't download 'webvfx-1.2.0.txz'. Aborting.
 * Fetch failed for 'media-libs/webvfx-1.2.0', Log file:
 *  '/var/tmp/portage/media-libs/webvfx-1.2.0/temp/build.log'

>>> Failed to emerge media-libs/webvfx-1.2.0, Log file:
```

Reply:
And made that typo in both ebuilds, live and learn.